subprojects/launcher/src/main/java/org/gradle/tooling/internal/provider/continuous/package-info.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Contains classes handling continuous build execution and triggering.
```

Comment:
Meta: packages will always contain classes, so prefixing the Javadoc with that does not add anything.

Reply:
So you'd suggest something like this?
```
Continuous build execution and triggering.
```

Reply:
That'd be good, yes.